### DISCUSSION TITLE
How to install netbox-routeros Plugin?

### DISCUSSION THREAD

**Discussion by new0ne** [netbox-community/netbox#15188] [Help Wanted!]
Hi there!
I am wondering how i can install the "netbox-routeros" plugin in netbox?
Its linked on the official website:
https://netbox.dev/plugins/netbox-routeros/
...but there are no installation instructions to be found.
I`ve just installed netbox and would like to pull config from my routeros devices in my home-lab, looks like netbox-routeros "could" be the solution for that...
Would be great if somebody could help :)

Have a nice weekend

**Comment by elico** [comment]
@new0ne Did you managed to install it?
